src/checks/zcl_aoc_check_31.clas.abap

```diff
@@ -131,6 +136,11 @@ CLASS ZCL_AOC_CHECK_31 IMPLEMENTATION.
 
   METHOD constructor.
 
+    DATA lt_slin_desc_t     TYPE STANDARD TABLE OF slin_desc_t WITH EMPTY KEY.
```

Comment:
This will result in issues with 7.31 systems: EMPTY KEY is a language element introduced in NW7.40